puppet-lint.gemspec

```diff
@@ -2,12 +2,14 @@ $LOAD_PATH.push(File.expand_path('lib', __dir__))
 require 'puppet-lint/version'
 
 Gem::Specification.new do |spec|
-  spec.name = 'puppetlabs-puppet-lint'
+  spec.name = 'puppet-lint'
```

Comment:
This was merged quite fast. I would have suggested to revert the original commit instead. Or at least do the change in a dedicated PR and not the release PR. I think the renaming is a quite significant change and now it won't appear in the changelog?